### DISCUSSION TITLE
How can we find out the IDs of the icons that we can use from the sprite?

### DISCUSSION THREAD

**Discussion by MikeKronewald** [eviden-parallel/community#6] [Q&A] [answered]
In our Projekt we want to integrate Atos Parallel Icon. But there is a problem finding the correct IDs for the Icons. Is there a Documentation or link where all Icons with IDs are listed?

**Comment by Th3S4mur41** [comment, marked as answer]
Hi @MikeKronewald,
Thanks for your input... We will add [a way to find the icons to our documentation](https://github.com/atos-parallel/icons/issues/94).

In the meantime, since we are currently using phosphor icons, you can use [the phosphor icon search](https://phosphoricons.com/#toolbar) to find the icons you are looking for

**Reply by Th3S4mur41** [reply]
Thanks to @nathania-haryanto, Icons are now documented in [Storybook](https://main--62751021a57ceb004a5a5182.chromatic.com/?path=/story/icons-icons--page)
